src/web/blueprints/__init__.py

```diff
@@ -1,2 +1,3 @@
 
-from .landing import create_blueprint
+from .api import api
+from .landing import landing
```

Comment:
I think these relative imports are made absolute in your refactor tests pr.

Reply:
So far I've left the imports that start with `.` alone, since they seem to resolve okay. If they have trouble with `pyinstaller` though we should change them.